### ISSUE TITLE
Catchment attributes

### ISSUE THREAD

**Issue by jsadler2** [jzwart/delaware-water-temp#3]
I am gathering catchment attributes for the PRMS HRU's from https://wwwbrr.cr.usgs.gov/projects/SW_MoWS/GeospatialFabric.html

**Comment by jsadler2** [comment]
It's been going swimmingly until I realized I have this issue: I don't know how to relate the National HRU ids (what is in the subset model) to the HRU ids in the catchment attr data files which are region-specific

**Comment by jsadler2** [comment]
Here's a sample of the subset HRU ids - they have nationwide-ids that are used in the NHM:
![image](https://user-images.githubusercontent.com/6943441/72547756-d7f73e80-3852-11ea-8dad-4cdf6cbc1c1f.png)

**Comment by jsadler2** [comment]
And here are the regional HRU id's with the attributes (the max id number is 4827 in region 02):
![image](https://user-images.githubusercontent.com/6943441/72547854-13920880-3853-11ea-8657-7fa558e42c0a.png)

**Comment by jsadler2** [comment]
So I guess the question is: is there a way to relate the nationwide id's (the model) to the region-specific id's (what we have the attributes for)

**Comment by jsadler2** [comment]
ping @jzwart , @aappling-usgs

**Comment by aappling-usgs** [comment]
Yes! There's a shapefile or similar that has both...let me dig around.

**Comment by aappling-usgs** [comment]
GeospatialFabric_National.gdb (from https://www.sciencebase.gov/catalog/item/5362b683e4b0c409c6289bf6) has a layer 'nhruNationalIdentifier' that has columns for hru_id_nat (national ID) and hru_id (regional ID). There are also columns for hru_id_local (seems to be identical to hru_id) and hru_segment and hru_segment_orig (could be important for linking to segments?).

**Comment by jsadler2** [comment]
Awesome. Thanks, @aappling-usgs
